packages/orbit-components/src/Separator/index.tsx

```diff
@@ -31,43 +31,43 @@ const enum Indent {
+      [Indent.EIGHT_HUNDRED]: "w-[calc(100%_-_theme(spacing.800))]",
+      [Indent.XLarge]: "w-[calc(100%_-_theme(spacing.800))]", // deprecated
+      [Indent.ONE_THOUSAND]: "w-[calc(100%_-_theme(spacing.1000))]",
+      [Indent.XXLarge]: "w-[calc(100%_-_theme(spacing.1000))]", // deprecated
     },
     right: {
-      [Indent.none]: "ps-0",
-      [Indent.THREE_HUNDRED]: "ps-300",
-      [Indent.small]: "ps-300", // deprecated
-      [Indent.FOUR_HUNDRED]: "ps-400",
-      [Indent.medium]: "ps-400", // deprecated
-      [Indent.SIX_HUNDRED]: "ps-600",
-      [Indent.large]: "ps-600", // deprecated
-      [Indent.EIGHT_HUNDRED]: "ps-800",
-      [Indent.XLarge]: "ps-800", // deprecated
-      [Indent.ONE_THOUSAND]: "ps-1000",
-      [Indent.XXLarge]: "ps-1000", // deprecated
+      [Indent.none]: "w-full",
+      [Indent.THREE_HUNDRED]: "ms-300 w-[calc(100%_-_theme(spacing.300))]",
+      [Indent.small]: "ms-300 w-[calc(100%_-_theme(spacing.300))]", // deprecated
+      [Indent.FOUR_HUNDRED]: "ms-400 w-[calc(100%_-_theme(spacing.400))]",
+      [Indent.medium]: "ms-400 w-[calc(100%_-_theme(spacing.400))]", // deprecated
+      [Indent.SIX_HUNDRED]: "ms-600 w-[calc(100%_-_theme(spacing.600))]",
+      [Indent.large]: "ms-600 w-[calc(100%_-_theme(spacing.600))]", // deprecated
+      [Indent.EIGHT_HUNDRED]: "ms-800 w-[calc(100%_-_theme(spacing.800))]",
+      [Indent.XLarge]: "ms-800 w-[calc(100%_-_theme(spacing.800))]", // deprecated
+      [Indent.ONE_THOUSAND]: "ms-1000 w-[calc(100%_-_theme(spacing.1000))]",
+      [Indent.XXLarge]: "ms-1000 w-[calc(100%_-_theme(spacing.1000))]", // deprecated
     },
     center: {
-      [Indent.none]: "px-0",
-      [Indent.THREE_HUNDRED]: "px-300",
-      [Indent.small]: "px-300", // deprecated
-      [Indent.FOUR_HUNDRED]: "px-400",
-      [Indent.medium]: "px-400", // deprecated
-      [Indent.SIX_HUNDRED]: "px-600",
-      [Indent.large]: "px-600", // deprecated
-      [Indent.EIGHT_HUNDRED]: "px-800",
-      [Indent.XLarge]: "px-800", // deprecated
-      [Indent.ONE_THOUSAND]: "px-1000",
-      [Indent.XXLarge]: "px-1000", // deprecated
+      [Indent.none]: "w-full",
+      [Indent.THREE_HUNDRED]: "ms-300 w-[calc(100%_-_theme(spacing.300)*2)]",
+      [Indent.small]: "ms-300 w-[calc(100%_-_theme(spacing.300)*2)]", // deprecated
+      [Indent.FOUR_HUNDRED]: "ms-400 w-[calc(100%_-_theme(spacing.400)*2)]",
+      [Indent.medium]: "ms-400 w-[calc(100%_-_theme(spacing.400)*2)]", // deprecated
+      [Indent.SIX_HUNDRED]: "ms-600 w-[calc(100%_-_theme(spacing.600)*2)]",
+      [Indent.large]: "ms-600 w-[calc(100%_-_theme(spacing.600)*2)]", // deprecated
+      [Indent.EIGHT_HUNDRED]: "ms-800 w-[calc(100%_-_theme(spacing.800)*2)]",
+      [Indent.XLarge]: "ms-800 w-[calc(100%_-_theme(spacing.800)*2)]", // deprecated
+      [Indent.ONE_THOUSAND]: "ms-1000 w-[calc(100%_-_theme(spacing.1000)*2)]",
+      [Indent.XXLarge]: "ms-1000 w-[calc(100%_-_theme(spacing.1000)*2)]", // deprecated
```

Comment:
SideOffset & align for RTL version is not cooperating well:
- for `left | right` align, I get the offset on the right side

<img width="810" alt="image" src="https://github.com/user-attachments/assets/ec17554b-f5e1-4643-8c15-b563ee110c07">

- for `center` align, I get doubled-offset on the right side

<img width="799" alt="image" src="https://github.com/user-attachments/assets/8183d8a5-4d60-4fe7-a00a-2aacbca8a380">